packages/api/package.json

```diff
@@ -25,12 +25,18 @@
   },
   "homepage": "https://github.com/GoogleCloudPlatform/flaky-service#readme",
   "dependencies": {
+    "@google-cloud/connect-firestore": "^1.0.6",
     "@google-cloud/firestore": "^3.8.4",
     "body-parser": "^1.19.0",
     "cors": "^2.8.5",
     "express": "^4.17.1",
+    "express-session": "^1.17.1",
+    "mocked-env": "^1.3.2",
+    "moment": "^2.27.0",
+    "querystring": "^0.2.0",
```

Comment:
`querystring` is built into Node.js, like `url`:

https://nodejs.org/api/querystring.html

There should be no need for the dependency.